src/write_triangle_mesh.cpp

```diff
@@ -41,15 +42,20 @@ npe_arg(v, dense_float, dense_double)
 npe_arg(f, dense_int, dense_long, dense_longlong)
 npe_default_arg(force_ascii, bool, bool(true))
 
-
-
 npe_begin_code()
   assert_valid_3d_tri_mesh(v, f);
   // TODO: remove __copy
   //copy is necessary for the ply library
   Eigen::MatrixXi f_copy = f.template cast<int>();
   Eigen::MatrixXd v_copy = v.template cast<double>();
-  bool ok = igl::write_triangle_mesh(str, v_copy, f_copy, force_ascii);
+  
+  igl::FileEncoding encoding = igl::FileEncoding::Ascii;
+  if (!force_ascii) {
```

Comment:
"write_triangle_mesh" no longer has parameter "force_ascii" instead it takes in the enum FileEncoding. To not break old usage of our wrapper I simply assume if they are not forcing ascii, then they are forcing binary.

Reply:
A better method would be to update our API to also use the new enum... but that is a breaking change 👎 

PR that lead to this here: https://github.com/libigl/libigl/pull/1619

Reply:
i think in python is more clear, you can always to `force_ascii=True`
let's keep it as is